server/utils/validatorHelpers.js

```diff
@@ -112,7 +112,7 @@ class ValidatorHelpers {
    */
   static checkNumber(route, ...input) {
     input.forEach((element) => {
-      route.push(check(element).trim().isNumeric().withMessage(`please put in a valid number as ${element}`));
```

Comment:
why do you have to change this on this branch?

Reply:
that was a mistake, I was testing out something, but did not return it to it's original state correctly, what should I do next?